test/tests/default_block_check/diffusion_with_null_material_kernel_coverage_override.i

```diff
@@ -0,0 +1,128 @@
+[Problem]
+  kernel_coverage_check = ONLY_LIST
+  kernel_coverage_block_list = '0 1'
+[]
+
+[GlobalParams]
+  block = '0 1 3'
+[]
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 48
+    ny = 32
```

Comment:
all these tests are a little excessive in nx and ny.
in debug mode they are clocking at 3.5s which is above our 2s recommendation